ambassador.patch

```diff
@@ -0,0 +1,205 @@
+diff --git a/ambassador/src/main/java/com/iluwatar/ambassador/RemoteService.java b/ambassador/src/main/java/com/iluwatar/ambassador/RemoteService.java
```

Comment:
I think this is an unintentional file added to commit. We should remove this.

Reply:
My bad. Will do that. Didn't realize. Thanks.